torchao/testing/utils.py

```diff
@@ -456,15 +458,15 @@ def _test_slice_and_copy_similar_to_vllm(self, config: AOBaseConfig):
             param = l.weight
             param_data = param.data
             param_data = param_data.narrow(output_dim, start_idx, shard_size)
-            orig_value = param_data.qdata[0][0]
+            orig_values = param_data.qdata[0]
```

Comment:
the original check failed by chance, the two tensors had the same value at [0][0].  Checking the first row is more resistant to chance.